ARCs/arc-0069.md

```diff
@@ -138,7 +138,7 @@ Clients **SHOULD** render an ASA's latest ARC69 metadata. Clients **MAY** render
 
 ##### Updating ARC69 metadata
 
-If an ASA has a manager address, then the manager **MAY** update an ASA's ARC69 metadata. To do so, the manager sends a new `acfg` transaction with the entire metadata represented as JSON in the transaction's `note` field.
+If an ASA has a manager address, then the manager **MAY** update an ASA's ARC69 metadata. To do so, the manager sends a new `acfg` transaction with the entire metadata represented as JSON in the transaction's `note` field. The metadata inserted in `note`**MAY** also be serialized using MessagePack (see: https://github.com/msgpack/msgpack/blob/master/spec.md).

```

Comment:
This is a breaking change, I would be against it

Reply:
The MessagePack choice would be an option (may) to optimize the 1000 bytes available for notes and therefore saving bytes for meaningful data. Besides, if I remember right, Algorand uses MessagePack to serialize and transmit the note.

Reply:
I'm against this breaking change